src/AutoRest.CSharp/build/CodeGeneration.targets

```diff
@@ -16,6 +16,13 @@
       use the following command line (remove the space between minus minus): dotnet msbuild /t:GenerateCode /p:AutoRestAdditionalParameters="- -interactive"
     -->
 
+    <!--
+      <CadlAdditionalOptions></CadlAdditionalOptions>
+     
+      Allows passing additional cadl command line arguments, for example to run special emitter 
+      use the following command line: dotnet msbuild /t:GenerateCode /p:CadlAdditionalOptions="@azure-tools/cadl-csharp.csharpGeneratorPath=/absolute/path/to/artifacts/bin/AutoRest.CSharp/Debug/net6.0/AutoRest.CSharp.dll"
```

Comment:
One question: how do we append multiple options using this new parameter?

Reply:
CadlAddiotnalOptions will accept multiple options  as one string and sperate by semicolon